requirements/pip.txt

```diff
@@ -89,7 +89,9 @@ django-formtools==2.1
 # https://github.com/rtfd/readthedocs.org/issues/3999
 docker==3.1.3
 
-django-textclassifier==1.0
+# We want v2.0 but it's not released yet
+# django-textclassifier==2.0
+git+https://github.com/humitos/django-textclassifier.git@admin-action-on-validator#egg=django-textclassifier
```

Comment:
We want all my PRs from django-textclassifier merged before merging this one into Read the Docs.